jquery.tmpl.js

```diff
@@ -7,7 +7,18 @@
  * Dual licensed under the MIT or GPL Version 2 licenses.
  * http://jquery.org/license
  */
-(function( jQuery, undefined ){
+(function (factory) {
+	if (typeof define === 'function' && define.amd) {
+		// AMD
+		define(['jquery'], factory);
+	} else if (typeof exports === 'object') {
```

Comment:
On the server (node.js) the DOM is  (generally) not available, and nor is jQuery, so  I think supporting CommonJS may be unnecessary. Do you agree?